ConTabs.Tests/Table-Basic.cs

```diff
@@ -44,6 +45,26 @@ public void TableObjectHasExpectedNumberOfColumns()
             table.Columns.Count.ShouldBe(4);
         }
 
+        [Test]
+        public void Table_GivenClassWithoutPublicProperties_ThrowsPublicPropertiesNotFoundException()
+        {
+            // Arrange
+            var listOfTestClasses = DataProvider.ListOfInvalidTestData();
+
+            // Act - using try-catch due to static builder pattern, which precludes use of Assert.Throws.
+            var message = string.Empty;
+            try
+            {
+                var table = Table<InvalidTestDataType>.Create(listOfTestClasses);
+            }
+            catch (PublicPropertiesNotFoundException e)
+            {
+                message = e.Message;
+            }
+            
+            // Assert
+            Assert.AreEqual(message, "On Table<T> creation, no valid properties were identified. Check access modifiers.");
```

Comment:
Is testing for the exceptions _message_ unduly brittle? Could we test for the _type_ of exception instead?